src/exchanges/gmocoin-client.js

```diff
@@ -0,0 +1,175 @@
+const BasicClient = require("../basic-client");
+const { throttle } = require("../flowcontrol/throttle");
+const Ticker = require("../ticker");
+const Trade = require("../trade");
+const Level2Point = require("../level2-point");
+const Level2Snapshot = require("../level2-snapshot");
+const moment = require("moment");
+
+class GMOCoinClient extends BasicClient {
+  constructor({
+    wssPath = "wss://api.coin.z.com/ws/public/v1",
+    throttleMs = 1000,
+    watcherMs,
+  } = {}) {
+    super(wssPath, "GMOCoin", undefined, watcherMs);
+
+    this.hasTickers = true;
+    this.hasTrades = true;
+    this.hasLevel2Snapshots = true;
+    this._send = throttle(this._send.bind(this), throttleMs);
+  }
+
```

Comment:
Need to override `_onClosing` and clean up `this._send` so it doesn't fire if the socket is closed. Refer to https://github.com/altangent/ccxws/blob/master/CONTRIBUTING.md#rate-limits

```
  _onClosing() {
    this._send.cancel();
    super._onClosing();
  }
```